lerna.json

```diff
@@ -2,9 +2,7 @@
   "$schema": "node_modules/lerna/schemas/lerna-schema.json",
   "changelogPreset": "@okta/conventional-changelog-odyssey",
   "npmClient": "yarn",
-  "packages": [
-    "packages/*"
-  ],
```

Comment:
Prettier wasn't run, and yet this was committed somehow. I've fixed it in my PR because of that.